chainercv/visualizations/vis_label.py

```diff
@@ -0,0 +1,42 @@
+def vis_label(label, alpha=1, ax=None):
+    """Visualize label of semantic segmentation.
+
+    Example:
+
+        >>> from chainercv.datasets import VOCSemanticSegmentationDataset
+        >>> from chainercv.visualizations import vis_image
+        >>> from chainercv.visualizations import vis_label
+        >>> import matplotlib.pyplot as plot
+        >>> dataset = VOCSemanticSegmentationDataset()
+        >>> img, label = dataset[60]
+        >>> ax = vis_image(img)
+        >>> vis_label(label, alpha=0.75, ax=ax)
+        >>> plot.show()
+
+    Args:
+        label (~numpy.ndarray): An integer array of shape
+            :math:`(height, width)`.
+            The values correspond to id for label names stored in
+            :obj:`label_names`.
```

Comment:
label_names are not given.
Perhaps, simply put it as "The values corresponds to classes assigned to pixels"?